problems/strings.js

```diff
@@ -43,7 +53,11 @@ let amount = 1000000
 * @param {number}
 * @returns {string}
  */
-function drEvil() {}
+function drEvil(num) {
+  if (num >= 1000000) {
+    return num + " dollars (pinky)";
+  } else return num + " dollars";
```

Comment:
Once you write else, you should immediately have curly braces {}. The curly braces will include your code block for the else statement.